packages/.gitignore

```diff
@@ -1,5 +1,7 @@
 coverage
+build
```

Comment:
Do we still need this build?

Reply:
Not at the moment. We can add it if required later.